pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -322,13 +324,6 @@ func (e *cloudWatchExecutor) executeLogAlertQuery(ctx context.Context, req *back
 			return nil, err
 		}
 
-		result, err := e.executeStartQuery(ctx, logsClient, model, q.TimeRange)
```

Comment:
do you mind talking through this one more? Why did we have it/why remove it?

Reply:
the alertQuery function below calls executeStartQuery on the same model and replaces the QueryId with the one from that response, so there's not point to calling it here

Reply:
ah gotcha! thanks!